app/presenters/return-logs/setup/readings.presenter.js

```diff
@@ -58,8 +58,22 @@ function _inputLines(lines) {
   })
 }
 
+function _lineLabel(endDate, returnsFrequency) {
+  const labelDate = new Date(endDate)
+
+  if (returnsFrequency === 'week') {
+    return `Week ending ${formatLongDate(labelDate)}`
+  }
+
+  if (returnsFrequency === 'month') {
+    return labelDate.toLocaleDateString('en-GB', { month: 'long', year: 'numeric' })
+  }
+
+  return formatLongDate(labelDate)
+}
+
 function _pageTitle(date) {
-  const titleDate = date.toLocaleDateString('en-GB', { year: 'numeric', month: 'long' })
+  const titleDate = date.toLocaleDateString('en-GB', { month: 'long', year: 'numeric' })
```

Comment:
We already have a helper for doing this in the `BasePresenter`, which you were previously using. Could we update this to use the existing helper?

```suggestion
  const titleDate = formatLongDate(date)
```

Reply:
This one is a bit different, it just displays the month and year and drops the day.

As discussed will move it into the base presenter.